ts/features/wallet/onboarding/cobadge/screens/start/CoBadgeChosenBankScreen.tsx

```diff
@@ -24,15 +31,40 @@ const tos_url =
 
 const partecipatingBank_url =
```

Comment:
```suggestion
const participatingBank_url =
```

Reply:
Fixed in [66cac06](https://github.com/pagopa/io-app/pull/2940/commits/66cac063ac2a2dd6a21a9e9287d258e426da1e1f)